meson.build

```diff
@@ -39,6 +38,12 @@ else
   error('Cannot found header mustache.hpp')
 endif
 
+libzim_dep = dependency('libzim', version : '>=6.3.0', static:static_deps)
+if not compiler.has_header_symbol('zim/zim.h', 'LIBZIM_WITH_XAPIAN')
```

Comment:
Shouldn't `zim/zim_config.h` be checked instead?

Reply:
@mgautierfr ?

Reply:
`zim.h` includes `zim_config.h`.
Both header file would work but I prefer to keep using `zim.h` as I tend to consider `zim_config.h` as a bit more internal.

This is really a matter of opinion here. There is no real separation (and documentation) of what can be directly including or not.